### PR TITLE
DIV-6619: release general order

### PR DESCRIPTION
**Pull request by yarpo**
Sub-task:
https://tools.hmcts.net/jira/browse/DIV-6619

Release ticket:
https://tools.hmcts.net/jira/browse/DIV-6497

### CODE REVIEW

**Review by madjava — APPROVED** [review]

File: test/unit/definitions/divorce/CaseTypeTab.test.js [hunk 1]
```diff
@@ -71,65 +70,12 @@ describe('CaseTypeTab (nonprod)', () => {
   });
 });
 
-/**
- * This suite (and sub suites) checks to make sure that there are no duplicate TabDisplayOrder or
- * TabFieldDisplayOrder in the concatenated release definition.
- * Note: As a post release activity the related release describe block can be removed
- * as the feature file would no longer exist
- */
-describe('CaseTypeTab duplicate checks (nonprod)', () => {
```
Comment (jvictorcf) [inline test/unit/definitions/divorce/CaseTypeTab.test.js:80]: Why are you removing this?

File: test/unit/utils/caseTabTypeHelper.js [hunk 2]
```diff
@@ -58,6 +58,7 @@ const nonProdTabIds = [
   // 'SolicitorCoRespondent', - TODO - uncomment this when we go live with AOS pack 2
```
Comment (jvictorcf) [inline test/unit/utils/caseTabTypeHelper.js:58]: Could we uncomment this?

**Review by jvictorcf — COMMENTED** [review]
Just a couple of comments.

File: test/unit/definitions/divorce/CaseTypeTab.test.js (hunk 1 above)
Comment (yarpo) [reply test/unit/definitions/divorce/CaseTypeTab.test.js:80]: I don't see any sense for it. 

`Prod` and `non-prod`. There is no point of testing every single "release" separately.

File: test/unit/utils/caseTabTypeHelper.js (hunk 2 above)
Comment (yarpo) [reply test/unit/utils/caseTabTypeHelper.js:58]: I have no idea why we have this file.

If it's still commented out apparently we don't need it.

**Review by jvictorcf — APPROVED** [review]

**Pull request merged**